src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -492,50 +492,61 @@ private BoundStatement BindLocalFunctionStatement(LocalFunctionStatementSyntax n
             var hasErrors = localSymbol.ScopeBinder
                 .ValidateDeclarationNameConflictsInScope(localSymbol, diagnostics);
 
-            BoundBlock block;
+            BoundBlock blockBody = null;
+            BoundBlock expressionBody = null;
             if (node.Body != null)
             {
-                block = BindEmbeddedBlock(node.Body, diagnostics);
+                blockBody = runAnalysis(BindEmbeddedBlock(node.Body, diagnostics), diagnostics);
+
+                if (node.ExpressionBody != null)
+                {
+                    var expressionBodyDiagnostics = new DiagnosticBag();
+                    expressionBody = runAnalysis(BindExpressionBodyAsBlock(node.ExpressionBody, expressionBodyDiagnostics), expressionBodyDiagnostics);
```

Comment:
Why drop these errors rather than reporting them?

Reply:
For now, I'm maintaining existing semantics here, which is to not report errors. I'll file a bug to follow up on this and consider reporting errors inside both bodies.

---
In reply to: [178338837](https://github.com/dotnet/roslyn/pull/25782#discussion_r178338837) [](ancestors = 178338837)

Reply:
Filed [https://github.com/dotnet/roslyn/issues/25845](https://github.com/dotnet/roslyn/issues/25845).

---
In reply to: [178372391](https://github.com/dotnet/roslyn/pull/25782#discussion_r178372391) [](ancestors = 178372391,178338837)